applications.yaml

```diff
@@ -1498,18 +1468,11 @@
     id: zen.exe
     matching_strategy: Equals
   options:
-  - object_name_change
-  - tray_and_multi_window
+  - force
   float_identifiers:
-  - kind: Class
-    id: MozillaTaskbarPreviewClass
-    matching_strategy: Legacy
-  - - kind: Title
-      id: Picture-in-Picture
-      matching_strategy: Equals
-    - kind: Exe
-      id: zen.exe
-      matching_strategy: Equals
+  - kind: Exe
```

Comment:
What the is the reason for adding a float rule to target every browser window?

Reply:
Truthfully I'm new to Komorebi and mimicked what other applications are doing, I tested without the float rule and it works fine so will remove.